### ISSUE TITLE
restnoded restarts immediately after posting the decleartion

### ISSUE THREAD

**Issue by himalayanBuddy** [F5Networks/f5-appsvcs-extension#232]
### Environment
 * Application Services Version:
{
    "version": "3.17.1",
    "release": "1",
    "schemaCurrent": "3.17.0",
    "schemaMinimum": "3.0.0"
}
 * BIG-IP Version:
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1.5
  Build       0.0.4
  Edition     Point Release 5
  Date        Wed Apr 24 21:26:13 PDT 2019

### Summary

After sending post request to URI <BIG-IP IP address>/mgmt/shared/appsvcs/declare, restnoded restarts.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
     "class": "AS3",
     "action": "deploy",
     "persist": true,
     "targetHost": "{{targetHost}}",
     "targetTokens": {
     "X-F5-Auth-Token": "{{authToken}}"
     },
     "declaration": {
         "class": "ADC",
         "schemaVersion": "3.17.0",
         "id": "simple test 1",
         "label": "sample_1",
         "remark": "Simple HTTP Service with Round-Robin Load Balancing",
         "updateMode": "selective",
         "Sample_01": {
             "class": "Tenant",
             "defaultRouteDomain": 0,
             "A1": {
                 "class": "Application",
                 "template": "http",
                 "serviceMain": {
                     "class": "Service_HTTP",
                     "virtualAddresses": [
                         "10.0.1.10"
                     ],
                     "pool": "web_pool"
                 },
                 "web_pool": {
                     "class": "Pool",
                     "monitors": [
                         "http"
                     ],
                     "members": [
                         {
                             "servicePort": 80,
                             "serverAddresses": [
                                 "192.0.1.10",
                                 "192.0.1.11"
                             ]
                         }
                     ]
                 }
             }
         }
     }
 }
 ```

2. Observe the following error response:
```json
 {
    "error": {
        "code": 404,
        "message": "",
        "innererror": {
            "referer": "restnoded",
            "originalRequestBody": "{xxxxx}",
            "errorStack": []
        }
    }
}
```

### Expected Behavior
Partition, virtual server and pool member created.

### Actual Behavior
Received 404 error message

/var/log/audit
status=[01020036:3: The requested value list (/Common/appsvcs/____appsvcs_lock) was not found.] cmd_data=delete ltm data-group internal /Common/appsvcs/____appsvcs_lock

/var/log/ltm
 01020036:3: The requested value list (/Common/ServiceDiscovery/tasks) was not found.
 The requested value list (/Common/appsvcs/____appsvcs_lock) was not found.

/var/log/restnoded
TypeError: Cannot read property 'code' of null
,    at /var/config/rest/iapps/f5-appsvcs/lib/util/networkUtil.js:44:50
,    at GetAddrInfoReqWrap.asyncCallback [as callback] (dns.js:64:16)
,    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:83:10)

Tue, 18 Feb 2020 18:08:35 GMT - info: path /var/config/rest/nodejskeystore/ is not exist
Tue, 18 Feb 2020 18:08:35 GMT - severe: [RestOperationDispatcher] 'shared/appsvcs/declare' not found.

**Comment by dstokesf5** [comment]
We have a similar issue in our internal product backlog (AUTOTOOL-1065), however the bug has been appearing very infrequently and intermittently. Does this failure happen consistently for you?

**Comment by himalayanBuddy** [comment]
Yes it's consistent for me.

**Comment by dstokesf5** [comment]
AUTOTOOL-1065 was resolved in version 3.18.0. Closing.
